gradlew.bat

```diff
@@ -84,6 +84,7 @@ set CMD_LINE_ARGS=%*
 
 set CLASSPATH=%APP_HOME%\gradle\wrapper\gradle-wrapper.jar
 
+

```

Comment:
is the extra white space necessary? IMHO it could be removed, but not too worried.

Reply:
auto generated by gradle wrapper